tools/iqtree.md

```diff
@@ -22,16 +22,16 @@ Using the SRA number as the main identifier of each obelisk sample sequence.
 eg."(OBELISK_SRA_NUMBER) >
 obelisk_sequence""
 Then I saved the file with a ".txt" suffix.
-![Alt text](/Users/caitlinoxley/Downloads/tutorial_q1.png)
+![Alt text](https://github.com/ababaian/VIRUSxDISCVRY/blob/main/tools/img/iqtree/tutorial_q1.png)
```

Comment:
Very close! Great job. You can actually simplify this to "point" from the place where this file lives. so change this to 
```
![Make this a better description](img/iqtree/tutorial_q1.png)
```